apps/io-func-wallet-solution-backend/src/infra/azure/cosmos/nonce.ts

```diff
@@ -19,4 +19,19 @@ export class CosmosDbNonceRepository implements NonceRepository {
       (error) => new Error(`Error inserting nonce: ${error}`)
     );
   }
+
+  delete(nonce: string) {
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏Please add docblock to every public method and an explicit returning type. This helps users to understand how to use the function without the need of understanding its internals

Reply:
I don't think a docblock is necessary since the methods are very simple, but I think I could add a comment regarding the `404` part